src/diffusers/pipelines/marigold/pipeline_marigold_depth.py

```diff
@@ -0,0 +1,1246 @@
+                        denoising_steps,
+                        ensemble_size,
+                        processing_resolution,
+                        match_input_resolution,
+                        resample_method_input,
+                        resample_method_output,
+                        batch_size,
+                        check_input,
+                        ensembling_kwargs,
+                        output_prediction_format,
+                        output_visualization,
+                        output_visualization_kwargs,
+                        output_uncertainty,
+                        output_latent,
+                        progress_bar,
+                    )
+                )
+
+        if len(out) == 1:
+            out = out[0]
+        else:
+            prediction = [o.prediction for o in out]
+            visualization = [o.visualization for o in out] if output_visualization else None
+            uncertainty = [o.uncertainty for o in out] if output_uncertainty else None
+            latent = [o.latent for o in out] if output_latent else None
+            if input_image_stacked:
+                if output_prediction_format == "np":
+                    prediction = np.stack(prediction)
+                    if uncertainty is not None:
+                        uncertainty = np.stack(uncertainty)
+                elif output_prediction_format == "pt":
+                    prediction = torch.stack(prediction)
+                    if uncertainty is not None:
+                        uncertainty = torch.stack(uncertainty)
+                if latent is not None:
+                    latent = torch.stack(latent)
+            out = MarigoldDepthOutput(
+                prediction=prediction, visualization=visualization, uncertainty=uncertainty, latent=latent
+            )
+
+        config = dict(  # noqa: C408
+            scheduler=scheduler,
+            preset=preset,
+            denoising_steps=denoising_steps,
+            ensemble_size=ensemble_size,
+            processing_resolution=processing_resolution,
+            match_input_resolution=match_input_resolution,
+            resample_method_input=resample_method_input,
+            resample_method_output=resample_method_output,
+            batch_size=batch_size,
+            save_memory=save_memory,
+            check_input=check_input,
+            ensembling_kwargs=ensembling_kwargs,
+            input_latent=None if input_latent is None else "provided",
+            generator=None if generator is None else "provided",
+            output_prediction_format=output_prediction_format,
+            output_visualization=output_visualization,
+            output_visualization_kwargs=output_visualization_kwargs,
+            output_uncertainty=output_uncertainty,
+            output_latent=output_latent,
+        )
+        out.config = config
+        logger.info(json.dumps(config, indent=2))
```

Comment:
We don't it from the pipelines.

Reply:
Since this is a new kind of pipeline (dense regression/perception), I thought it might be a good idea to report anything resolved automatically in the code or from the checkpoint, especially: the scheduler, the number of denoising steps, ensemble_size, and such. Since you have pointed out above that there should be less implicit stuff, I agree to have this removed after fixing those issues.

Reply:
Fixed